### PR TITLE
[Doc] review migration

### PR DESCRIPTION
**Pull request by ly015**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by liqikai9** [comment]
I think we may add a catalog at the first of `migration.md`

File: docs/en/migration.md [hunk 1]
```diff
@@ -442,7 +466,7 @@ init_cfg=dict(
     checkpoint='torchvision://resnet50')
 ```
 
-In addition to these commonly used backbones, you can easily use backbones from repositories in the OpenMMLab ecosystem such as MMClassification, which all share the same config system and provide pre-trained weights.
```
Comment (ly015) [inline docs/en/migration.md:445]: `OpenMMLab ecosystem ` may be confused with https://github.com/open-mmlab/ecosystem

**Comment by ly015** [comment]
Currently, we use a mixture of 3rd person (`users`) and 2nd person ('you') in the documents. Should we unify to either one of them?

**Comment by Tau-J** [comment]
> 

well, I think it doesn't matter. We can modify later if need

**Review by Tau-J — APPROVED** [review]

**Pull request merged**